back/spec/policies/initiative_policy_spec.rb

```diff
@@ -327,4 +327,24 @@
       end
     end
   end
+
+  context 'when initiative has an associated cosponsor' do
+    let!(:initiative) do
+      create(:initiative, initiative_status: create(:initiative_status_proposed))
+    end
+    let(:cosponsor) { create(:user) }
+    let!(:cosponsors_initiative) { create(:cosponsors_initiative, initiative: initiative, user: cosponsor) }
+
+    context 'for a user who is cosponsor of the initiative' do
+      let(:user) { cosponsor }
+
+      it { is_expected.to permit(:accept_invite) }
+    end
+
+    context 'for a user who is NOT cosponsor of the initiative' do
+      let(:user) { create(:user) }
+
+      it { is_expected.not_to permit(:accept_invite) }
+    end
+  end
```

Comment:
Super simple tests for now.

Soon, we will incorporate cosponsors in the more detailed existing tests, when we work on [TAN-234](https://www.notion.so/citizenlab/BE-Include-cosponsors-in-initiative_policy-f1b59ca5051b4749bc4414d5bce2fde5?pvs=4) (high priority, and probably the next thing I work on)